cromwell-drs-localizer/src/test/scala/drs/localizer/downloaders/AccessUrlDownloaderSpec.scala

```diff
@@ -15,31 +14,45 @@ class AccessUrlDownloaderSpec extends AnyFlatSpec with CromwellTimeoutSpec with
 
     val downloader = AccessUrlDownloader(
       accessUrl = AccessUrl(url = fakeAccessUrl, headers = None),
-      downloadLoc = fakeDownloadLocation
+      downloadLoc = fakeDownloadLocation,
+      hashes = None
     )
 
-    val expected = s"""mkdir -p $$(dirname '$fakeDownloadLocation') && rm -f '$fakeDownloadLocation' && curl --silent --write-out '%{http_code}' --location --fail --output '$fakeDownloadLocation' '$fakeAccessUrl'"""
+    val expected = s"""mkdir -p $$(dirname '$fakeDownloadLocation') && rm -f '$fakeDownloadLocation' && getm --checksum-algorithm 'null' --checksum 'null' --filepath '$fakeDownloadLocation' '$fakeAccessUrl'"""
 
     downloader.generateDownloadScript() shouldBe expected
   }
 
-  val results: TableFor3[Int, String, DownloadResult] = Table(
-    ("exitCode", "httpStatus", "download result"),
-    (0, "200", DownloadSuccess),
-    (0, "408", RetryableDownloadFailure(ExitCode(0))),
-    (1, "408", RetryableDownloadFailure(ExitCode(1))),
-    (1, " foobar ", RetryableDownloadFailure(ExitCode(1))),
-    (1, "503", RetryableDownloadFailure(ExitCode(1))),
-    (1, "429", NonRetryableDownloadFailure(ExitCode(1))),
-    (7, "429", RetryableDownloadFailure(ExitCode(7))),
-    (0, "  ", RetryableDownloadFailure(ExitCode(0))),
-  )
-
-  val accessUrlDownloader: AccessUrlDownloader = AccessUrlDownloader(null, null)
-
-  forAll(results) { (exitCode, httpStatus, expected) =>
-    it should s"produce $expected for exitCode $exitCode and http status '$httpStatus'" in {
-      accessUrlDownloader.result(exitCode, httpStatus, "drs://foo/bar") shouldBe expected
+  {
```

Comment:
Nice job characterizing `getm`s output behavior! This will make it easy to upgrade with confidence in the future!